DOc/docs/source/Options/user-story-4.rst

```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
 
 User Story
 ----------
-What is Lorem Ipsum?
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
-orem Ipsum has been the industry's standard dummy text ever since the 1500s, 
-when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
-It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. 
-It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software 
-like Aldus PageMaker including versions of Lorem Ipsum.
-
-Why do we use it?
-It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. 
-The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', 
-making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 
-'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose 
-(injected humour and the like).
 
+Next to data governance, data quality is another core pillar of data management. 
+Good data quality helps make data more useful and usable, resulting in better business outcomes.
+Meanwhile, bad data quality limits the value of data and results in worse business outcomes. 
 
+Data quality can be measured in many ways. 
+It is the responsibility of the data governance organisation to define data quality metrics.
+Data quality is typically measured along the following dimensions:
+
+| Dimension    | Description                                                                                                             |
+|--------------|-------------------------------------------------------------------------------------------------------------------------|
+| Accuracy     | Whether or not the data contains errors. For example, a person's name should always be correct.                         |
+| Completeness | Whether or not each row of the data has a value. For example, a person should always have a name.                       |
+| Timeliness   | Whether or not the data is up to date. For example, the data should only include people currently employed.             |
+| Uniqueness   | Whether or not values that need to be unique, actually are unique. For example, every employee should have a unique ID. |
+| Validity     | Whether or not the data fits with a predefined format. For example, a person's initials should always be capitalized.   |
+
+For every dimension, one or more data quality rules can be defined.
+For every rule, a percentage score is assigned. 
```

Comment:
For every rule, a percentage score is calculated by dividing the number of compliant entities over the number of all entities,